components/dockmanager/events.md

```diff
@@ -48,19 +49,23 @@ The event handler receives as an argument an `DockManagerUndockEventArgs` object
 
 ## VisibleChanged
 
-The `VisibleChanged` event is fired when the user tries to hide a given pane. You can effectively cancel the event by not propagating the new visibility state to the variable the `Visible` property is bound to. This is the way to cancel the event and keep the pane visible.
+The `VisibleChanged` event fires when the user tries to hide a given pane. You can effectively cancel the event by not propagating the new visibility state to the variable the `Visible` property is bound to. This is the way to cancel the event and keep the pane visible.
 
 ## SizeChanged
 
-The `SizeChanged` event is triggered when the `Size` parameter of the corresponding pane is changed.
+The `SizeChanged` event fireswhen the `Size` parameter of the corresponding pane changes.
+
+## UnpinnedChanged
+
+The `UnpinnedChanged` event fireswhen the `Unpinned` parameter of the corresponding pane changes.
 
 ## UnpinnedSizeChanged
 
-The `UnpinnedSizeChanged` event is triggered when the `UnpinnedSize` parameter of the corresponding pane is changed.
+The `UnpinnedSizeChanged` event fires when the `UnpinnedSize` parameter of the corresponding pane changes.
 
 ## OnPaneResize
 
-The `OnPaneResize` event is fired when a pane is resized, except unpinned panes. It lets you respond to that change if needed - for example, call the `.Refresh()` method of a chart or otherwise repaint a child component in the content. You can also use it to, for example, update the saved [state](slug:dockmanager-state) for your users.
+The `OnPaneResize` event fires when a pane is resized, except unpinned panes. It lets you respond to that change if needed - for example, call the `.Refresh()` method of a chart or otherwise repaint a child component in the content. You can also use it to, for example, update the saved [state](slug:dockmanager-state) for your users.
```

Comment:
The first sentence sounds very unclear. I recommend revising it for clarity and better readability.